1.11/security/ent/perms-reference/index.md

```diff
@@ -56,34 +56,41 @@ By convention, `full` indicates that the permission supports all other action id
 
 ## <a name="mesos"></a>Mesos Permissions
 
+Permission string sections for `<role-name>`s in square brackets are optional.
+These sections allow the permission scope to be narrowed.
+When no role name is specified, the permissions apply to all roles.
```

Comment:
Can we say "apply to all Mesos roles" here?

In the context of security and permissions, the term "role" can easily be understood as "security role". We need to always carefully distinguish "Mesos roles" from that.